config/jetty.yml

```diff
@@ -2,5 +2,16 @@ default:
   startup_wait: 90
   jetty_port: 8983
   java_opts:
-    - "-Xmx512m"
-    - "-XX:MaxPermSize=128m"
+    - "-Xmx1024m"
+    - "-XX:NewSize=256m"
+    - "-XX:MaxNewSize=256m" 
+    - "-XX:MetaspaceSize=256m"
+    - "-XX:MaxMetaspaceSize=512m"
+    - "-XX:+DisableExplicitGC"
+    - "-Dfcrepo.audit.container=/audit"
+    - "-Dfcrepo.modeshape.index.directory=/var/www/sites/hydranorth/jetty/fcrepo4-data/fcrepo.index.directory"
+    - "-Dfcrepo.log.jcr=DEBUG"
+    - "-Dfcrepo.log.oai=DEBUG"
+    - "-Dfcrepo.log.maxHistory=30"
+    - "-Dfcrepo.log.totalSizeCap=3G"
+    - "-Dfcrepo.log.directory=/var/www/sites/hydranorth/jetty/logs" 
```

Comment:
There are a couple of lines that are specific to our deployment choices (/var/www/sites/hydranorth).  Maybe make this a template file in the ansible playbook?